### PR TITLE
style: design tweaks

### PR DESCRIPTION
**Pull request by aspencer116**
# Assorted visual changes
- Fixed SVG size issue on Android 4
- Added a favicon
- Adjusted the size of the SVG animals and the smaller background shapes
- Changed the style of the action/navigation bar
- Adjusted the brightness of the tote bag detail image
## Where to see the changes compared to master:
- In the "about" section (with the SVG animals) the animals and smaller shapes all get bigger on larger screens.
- The action/nav bar is now a yellow-green color as opposed to brown and orange
- Added a favicon for the site

### CODE REVIEW

File: docs/css/styles.css [hunk 1]
```diff
@@ -133,26 +133,33 @@
```
Comment (iAmNathanJ) [inline docs/css/styles.css:133]: This looks like compiled scss. Just curious, is there a reason for committing compiled assets to the repo? Or am I confused?

File: src/partials/_head.hbs [hunk 2]
```diff
@@ -2,6 +2,8 @@
 <meta charset="UTF-8">
 <meta name="author" content="Andrew Spencer">
 <meta name="viewport" content="width=device-width,initial-scale=1.0">
+<link rel="shortcut icon" type="image/png" href="images/AME_favicon.png">
+<link rel="icon" type="image/png" href="images/AME_favicon.png">
```
Comment (iAmNathanJ) [inline src/partials/_head.hbs:17]: This is a nice addition. This whole favicon thing is really painful, though. If you haven't already, check out https://github.com/joshbuchea/HEAD#favicons and the links there. That entire README has about all you could want to know about what goes inside `<head>`

File: src/scss/_action.scss [hunk 3]
```diff
@@ -34,58 +34,44 @@
     justify-content: center;
 
     //content unique to this instance of c-action__link
-    background-color: $orange;
+    background-color: darken($green-brand, 2%);
```
Comment (iAmNathanJ) [inline src/scss/_action.scss:37]: I love your choice here. Along with the header and new favicon, everything looks more cohesive.

Comment (aspencer116) [reply src/scss/_action.scss:37]: Thanks! And if you have any design or content suggestions feel free to let me know! Does anything appear "off" to you?

File: src/scss/_action.scss [hunk 4]
```diff
@@ -34,58 +34,44 @@
...
 
   &__arrow {
-    fill: $orange;
+    fill: darken($green-brand, 2%);
     width: 1.5em;
     position: absolute;
-    top: 52px;
-    top: 3.25rem;
+    top: 50px;
+    top: 3.2rem;
```
Comment (iAmNathanJ) [inline src/scss/_action.scss]: nitpick: should this be `3.125rem` to match `50px`?

File: src/scss/_svg.scss [hunk 5]
```diff
@@ -1,14 +1,16 @@
 .c-AMElogo__color {
   display: block;
-  max-width: 12.5em;
+  height: 3em;
+  width: 12.5em; //for Android 4.1.2
+  width: 12.5em;
```
Comment (iAmNathanJ) [inline src/scss/_svg.scss]: Is one of these supposed to be `max-width`?

File: src/scss/_svg.scss [hunk 6]
```diff
@@ -18,14 +20,20 @@
...
-  bottom: 4em; //if no javascript loads
+  bottom: 4em;
+  display: block;
+  left: 25%; //for browsers with no calc() support. Android 4.1.2
   left: calc(50% - 6.25em);
   z-index: 1;
 
   @media (min-width: $bp-e) {
     width: 18em;
     left: calc(50% - 9em);
```
Comment (iAmNathanJ) [inline src/scss/_svg.scss:36]: I see fallbacks for no `calc()` which is great. Should you provide one here as well?

**Comment by iAmNathanJ** [comment]
So awesome that you're using css animations. I noticed that you're prefixing a few of the declarations e.g. `@-webkit-keyframes`. Autoprefixer is handling that for you in the build, so every animation with the standard `@keyframes` is getting the standard _and_ prefixed versions in compilation. I would change them all to the non-prefixed version and let autoprefixer take care of the rest.

File: src/scss/_action.scss (hunk 3 above)
Comment (iAmNathanJ) [reply src/scss/_action.scss:37]: One spot that isn't sitting just right for me.

screen shot 2016-09-01 at 9 40 24 am

I wonder about the line height on these paragraphs. The headings are pretty tight on the copy but there's a lot of air in line height of the text. I might play with those distances a little bit.

Other than that, I really can't criticize the design. It looks great. Much better than I could do. 😄

**Pull request merged**